azuki-core/src/main/kotlin/com/anaplan/engineering/azuki/core/runner/TaskWrapper.kt

```diff
@@ -99,6 +108,8 @@ private class LogAndCaptureOutputStream(
     }
 
     override fun close() {
+        log(String(buffer.toByteArray()))
+        buffer.clear()
         capture.close()
     }
```

Comment:
I presume the same change should also be made in the Gradle plugin:
https://github.com/anaplan-engineering/vdm-gradle-plugin/blob/b04c65fac6227f77ec383347be54fcc74838227d/src/main/kotlin/com/anaplan/engineering/vdmgradleplugin/TypeChecker.kt#L123

Reply:
I guess but it's not urgent, we (anyone?) very rarely print without a new line these days

It was just part of my 'why am I only getting half a log line?' debug that was nothing to do with this